bigquery/main_test.go

```diff
@@ -414,19 +453,101 @@ func TestSendNotification(t *testing.T) {
 		wantErr: false,
 	}, {
 		name: "no timestamp build",
+		cfg: &notifiers.Config{
+			Spec: &notifiers.Spec{
+				Notification: &notifiers.Notification{
+					Filter: `build.build_trigger_id == "1234" `,
+					Delivery: map[string]interface{}{
+						"table": "projects/project_name/datasets/dataset_name/tables/valid",
+					},
+				},
+			},
+		},
 		build: &cbpb.Build{
 			ProjectId:      "Project ID",
 			Id:             "Build ID",
 			BuildTriggerId: "1234",
 			Status:         4,
 		},
 		wantErr: true,
+	}, {
+		name: "table is empty",
+		cfg: &notifiers.Config{
+			Spec: &notifiers.Spec{
+				Notification: &notifiers.Notification{
+					Filter: `build.build_trigger_id == "123e4567-e89b-12d3-a456-426614174000" `,
+					Delivery: map[string]interface{}{
+						"table": "projects/project_name/datasets/dataset_name/tables/empty",
+					},
+				},
+			},
+		},
+		build: &cbpb.Build{
+			ProjectId:      "Project ID",
+			Id:             "Build ID",
+			BuildTriggerId: "123e4567-e89b-12d3-a456-426614174000",
+			Status:         3,
+			CreateTime:     timestamppb.Now(),
+			StartTime:      timestamppb.Now(),
+			FinishTime:     timestamppb.Now(),
+			Tags:           []string{},
+			Options:        &cbpb.BuildOptions{Env: []string{}},
+		},
+		wantErr: false,
+	}, {
+		name: "schema not initialized",
+		cfg: &notifiers.Config{
+			Spec: &notifiers.Spec{
+				Notification: &notifiers.Notification{
+					Filter: `build.build_trigger_id == "123e4567-e89b-12d3-a456-426614174000" `,
+					Delivery: map[string]interface{}{
+						"table": "projects/project_name/datasets/dataset_name/tables/notinitialized",
+					},
+				},
+			},
+		},
+		build: &cbpb.Build{
+			ProjectId:      "Project ID",
+			Id:             "Build ID",
+			BuildTriggerId: "123e4567-e89b-12d3-a456-426614174000",
+			Status:         3,
+			CreateTime:     timestamppb.Now(),
+			StartTime:      timestamppb.Now(),
+			FinishTime:     timestamppb.Now(),
+			Tags:           []string{},
+			Options:        &cbpb.BuildOptions{Env: []string{}},
+		},
+		wantErr: false,
+	}, {
+		name: "table exists with initialized schema",
```

Comment:
So just to be clear, this is testing writing to an already-existing table that has "bad" schema? You may want to update the test case name to make that clearer.